src/components/application_manager/src/commands/hmi/on_app_unregistered_notification.cc

```diff
@@ -44,6 +44,12 @@ OnAppUnregisteredNotification::OnAppUnregisteredNotification(
 
 OnAppUnregisteredNotification::~OnAppUnregisteredNotification() {}
 
+bool OnAppUnregisteredNotification::Init() {
```

Comment:
@yang1070 will be deleted

Reply:
@yang1070 @LuxoftAKutsan 
By default every command/notification uses Init() method from base class which makes HMI to mobile and mobile to HMI application id replacement. Here method is overloaded intentionally to not do that since for this particular notification replacement is not needed.

Reply:
ok